test/test_regression_cli.py

```diff
@@ -46,7 +46,6 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir("./test/")
```

Comment:
The original reason for this line was to allow us to include the raw data for two of the regression test cases instead of downloading it. The retriever's default search looks in a couple of places including `./raw_data` to see if the data exists before downloading it, so this is moving the working directory into `test` so that it sees `./test/raw_data`. I'm not sure this is the best design, but that's why it was there.